Modules/System Tests/Environment Information/src/EnvironmentInformationTest.Codeunit.al

```diff
@@ -78,5 +86,18 @@ codeunit 135091 "Environment Information Test"
         // [Then] Should return false
         Assert.IsFalse(EnvironmentInformation.IsSaaS(), 'Testability should have dictacted a non- SaaS environment');
```

Comment:
This typo was made everywhere... I'm guessing 'dictated' is what was meant.

Reply:
I will fix it when I implement the rest.